.babelrc.js

```diff
@@ -0,0 +1,27 @@
+let envOpts = {
+  loose: true
+};
+
+module.exports = {
+  comments: false,
+  presets: [
+    [
+      require("babel-preset-env"), envOpts
```

Comment:
I think the `require` calls here are still optional, not sure if you had a specific motivation for them.